diff --git a/mini_warehouse/apps/products/forms.py b/mini_warehouse/apps/products/forms.py
index 4a405f3667555f6e7b0439e6d7130c34afa8f502..62487c21b8d55eb14d1dc767a8d704932a3a34c8 100644
--- a/mini_warehouse/apps/products/forms.py
+++ b/mini_warehouse/apps/products/forms.py
@@ -1,7 +1,15 @@
 from django import forms
+
 from .models import Product
 
+
 class ProductForm(forms.ModelForm):
     class Meta:
         model = Product
         fields = ['name', 'stock', 'purchase_price', 'sale_price']
+        widgets = {
+            'name': forms.TextInput(attrs={'class': 'form-control'}),
+            'stock': forms.NumberInput(attrs={'min': 0, 'class': 'form-control'}),
+            'purchase_price': forms.NumberInput(attrs={'min': 0, 'step': '0.01', 'class': 'form-control'}),
+            'sale_price': forms.NumberInput(attrs={'min': 0, 'step': '0.01', 'class': 'form-control'}),
+        }
